linux/lvm.go

```diff
@@ -114,10 +114,28 @@ func (ls *linuxLVM) ScanLVs(filter disko.LVFilter) (disko.LVSet, error) {
 	return lvs, nil
 }
 
-func (ls *linuxLVM) CreatePV(name string) (disko.PV, error) {
+func makeDevPathFromName(name string) string {
+	if strings.HasPrefix(name, "/dev/") {
+		return name
+	}
+
+	return "/dev/" + name
+}
+
+func makeDevNameFromPath(path string) string {
```

Comment:
this is a less sophisticated version of getKnameForBlockDevicePath .

there is a small difference in that `getKnameForBlockDevicePath` returns an error, insisting that the device exists. I think that its quite possible we may want such a method, but I don't think this use needs it (you're not going to be able to pvcreate something if it doesn't exist).